scss/_variables.scss

```diff
@@ -1266,12 +1266,13 @@ $form-switch-checked-bg-size:     add(map-get($spacers, 2), map-get($spacers, 1)
 // stylelint-disable-next-line function-disallowed-list
 $form-switch-checked-bg-position: calc(var(--#{$prefix}border-width) * 3) 50% !default; // Boosted mod
 
-$form-switch-checked-square-bg:     var(--#{$prefix}body-bg) !default; // Boosted mod
-$form-switch-checked-bg:            $supporting-orange !default; // Boosted mod
-$form-switch-checked-border-color:  $supporting-orange !default; // Boosted mod
-$form-switch-checked-filter:        none !default; // Boosted mod
-$form-switch-checked-focus-inner:   var(--#{$prefix}focus-visible-inner-color) !default; // Boosted mod
-$form-switch-checked-focus-outer:   var(--#{$prefix}focus-visible-outer-color) !default; // Boosted mod
+$form-switch-checked-square-bg:                 var(--#{$prefix}body-bg) !default; // Boosted mod
+$form-switch-checked-bg:                        $supporting-orange !default; // Boosted mod
+$form-switch-checked-border-color:              $supporting-orange !default; // Boosted mod
+$form-switch-checked-filter:                    none !default; // Boosted mod
+$form-switch-checked-focus-inner:               var(--#{$prefix}focus-visible-inner-color) !default; // Boosted mod
+$form-switch-checked-focus-outer:               var(--#{$prefix}focus-visible-outer-color) !default; // Boosted mod
+$form-switch-unchecked-invalid-border-color:  #31c3eb !default; // Boosted mod
```

Comment:
* Indentation issue
* Maybe add a comment after the "Boosted mod" to explain that with the filter, it gives the right color?
* This is a new Sass variable, so it should be referenced in the migration guide